install-pcf/gcp/pipeline.yml

```diff
@@ -276,6 +276,7 @@ jobs:
       db_silk_password: {{db_silk_password}}
       db_locket_username: {{db_locket_username}}
       db_locket_password: {{db_locket_password}}
+      BLOBSTORE_INTERNAL_ACCESS_RULES: {{blobstore_internal_access_rules}}
```

Comment:
This param also needs to be added to `pcf-pipelines/tasks/install-ert/configure-json/task.yml`, and used in `pcf-pipelines/tasks/install-ert/configure-json/task.sh`. Alternatively, remove this partly-implemented feature from the gcp and aws pipelines as only the vsphere pipeline fully implements it.